pilot/pkg/config/kube/ingress/controller.go

```diff
@@ -90,7 +90,7 @@ type controller struct {
 	services kclient.Client[*corev1.Service]
 }
 
-var IngressNamespace = env.Register("K8S_INGRESS_NS", constants.IstioIngressNamespace, "").Get()
+var IngressNamespace = env.Register("K8S_INGRESS_NS", constants.IstioSystemNamespace, "").Get()
```

Comment:
Not necessarily; the ingress gateway can be in a different namespace than the control plane (we recommend this now)

Reply:
Oh is this just setting a default value?

Reply:
Yes,  still allows custimization

Reply:
Do we want to continue with this default ? As we are making a lot of behavior changes when ambient
is enabled, and we have a mechanism to keep behavior for old versions - should we stop the dangerous
practice of running internet-exposed ingress and other things in the same namespace with the root CA private key ?

Reply:
I realized it's the namespace where the ingress controller runs - not where the ingress runs. Maybe - is it even possible to run the ingress controller in a different place from Istiod ? Maybe having a 'doc' string would help - assuming this can even be set to a different value than POD_NAMESPACE.

Reply:
Should add a doc.